src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -296,6 +316,16 @@ def test_WINNF_FT_S_PCR_1(self, config_filename):
     cbsd_ids = self.assertRegistered(config['registrationRequests'],
                                      config['conditionalRegistrationData'])
 
+    # Asserts the REG-Conditional value doesn't exist in the registrationRequest,
```

Comment:
Consider moving the PAL record injection *after* this block so that the ref model will catch PAL record errors (instead of the SAS UUT). It doesn't match the spec exactly but we are already deviating for the same reason.

Reply:
We   have already made necessary changes to reorder the way we trigger the PPA   reference model and capture any issue before trigger ppa creation admin api   to SAS UUT.     The way we have reordered mentioned as below:     1. assertRegConditionalsForPpaRefModel    method asserts and prepare the reqistration request by filling the   necessary  field required for PPA   reference model(This section has been already reviewed)     2. The below line will invoke ppa creation model before instruct SAS UUT to   trigger ppa boundary     ppa.PpaCreationModel(config['registrationRequests'],                                                            config['palRecords'])     3. Prepares ppa_creation_request for ppa_creation admin api and trigger PPA   creation to SAS UUT and waits for response and validate ppa creation at SAS   UUT is pass or fail based on     status check API . This will be accomplished by invoking the instance   method(This section has been already reviewed)         ppa_id =   self.triggerPpaCreationAndWaitUntilComplete(ppa_creation_request)          Let me know is there any change required re-order these API.?

Reply:
My suggestion was that you move `ppa.PpaCreationModel(...)` to before `self._sas_admin.InjectPalDatabaseRecord(...)` so that we catch any errors with the PAL DB records in the reference model, not in the SAS UUT. This applies to PCR.1 and PCR.2, not to PCR.3.

Reply:
Yes, I think it's OK to do assertRegistered() first. This looks good now. Thanks!